sdk/communication/azure-communication-chat/tests/_shared/helper.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import re
+import base64
+from azure_devtools.scenario_tests import RecordingProcessor
+from datetime import datetime, timedelta
+from functools import wraps
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
+import sys
+
+if sys.version_info[0] < 3 or sys.version_info[1] < 4:
+    # python version < 3.3
```

Comment:
We shouldn't need these checks any longer, currently supported Python versions are only 3.6 - 3.10